discoverer/src/main/java/eu/seaclouds/platform/discoverer/api/DiscovererAPI.java

```diff
@@ -0,0 +1,312 @@
+/**
+ * Copyright 2014 SeaClouds
+ * Contact: SeaClouds
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package eu.seaclouds.platform.discoverer.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import eu.seaclouds.platform.discoverer.core.Discoverer;
+import eu.seaclouds.platform.discoverer.core.Offering;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.Set;
+
+@Path("/")
+public class DiscovererAPI {
+
+    private final Discoverer discoverer;
+    private org.json.simple.parser.JSONParser JSONParser;
+    static Logger log = LoggerFactory.getLogger(DiscovererAPI.class);
+
+    public DiscovererAPI(Discoverer discoverer) {
+        this.discoverer = discoverer;
+        this.JSONParser = new JSONParser();
+    }
+
+    @POST
+    @Path("/delete")
+    @Produces(MediaType.APPLICATION_JSON)
+    public DeleteRepresentation deleteOfferingById(@QueryParam("oid") String offerId)
+            throws IOException {
+
+        ArrayList<String> removedOfferingIds = new ArrayList<>();
```

Comment:
why a list? I'm expecting `deleteOfferingById` to delete a single offering, so it should return a boolean, I think, surely not a list of a single ID.